VAMobile/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -112,7 +130,6 @@ export function HomeScreen({}: HomeScreenProps) {
     foldersData?.data.find((folder) => folder.attributes.name === FolderNameTypeConstants.inbox)?.attributes
       .unreadCount ||
     0
-  const { isLoading: loadingPersonalInfo } = usePersonalInformation()
```

Comment:
I removed this because it's not needed, since it's called in the `Nametag` component. Plus, it's not used for displaying any information on the Home screen